CreateSetupPackageLayout.proj

```diff
@@ -0,0 +1,97 @@
+    <!-- <BinX86Output Include="$(OutputPath)FileTracker.dll" /> -->
+    <!-- <BinX86Output Include="$(OutputPath)Microsoft.Build.Conversion.Core.dll" /> -->
+    <BinX86Output Include="$(OutputPath)Microsoft.Build.dll" />
+    <!-- <BinX86Output Include="$(OutputPath)Microsoft.Build.Engine.dll" /> -->
+    <BinX86Output Include="$(OutputPath)Microsoft.Build.Framework.dll" />
+    <!-- <BinX86Output Include="$(OutputPath)Microsoft.Build.Tasks.CodeAnalysis.dll" /> -->
+    <BinX86Output Include="$(OutputPath)Microsoft.Build.Tasks.Core.dll" />
+    <BinX86Output Include="$(OutputPath)Microsoft.Build.Utilities.Core.dll" />
+    <!-- <BinX86Output Include="$(OutputPath)Microsoft.CodeAnalysis.CSharp.dll" /> -->
+    <!-- <BinX86Output Include="$(OutputPath)Microsoft.CodeAnalysis.CSharp.Scripting.dll" /> -->
+    <!-- <BinX86Output Include="$(OutputPath)Microsoft.CodeAnalysis.dll" /> -->
+    <!-- <BinX86Output Include="$(OutputPath)Microsoft.CodeAnalysis.Scripting.dll" /> -->
+    <!-- <BinX86Output Include="$(OutputPath)Microsoft.CodeAnalysis.VisualBasic.dll" /> -->
+    <BinX86Output Include="$(OutputPath)Microsoft.Common.CurrentVersion.targets" />
+    <BinX86Output Include="$(OutputPath)Microsoft.Common.overridetasks" />
+    <BinX86Output Include="$(OutputPath)Microsoft.Common.targets" />
+    <BinX86Output Include="$(OutputPath)Microsoft.Common.tasks" />
+    <BinX86Output Include="$(OutputPath)Microsoft.CSharp.Core.targets" />
+    <BinX86Output Include="$(OutputPath)Microsoft.CSharp.CurrentVersion.targets" />
+    <BinX86Output Include="$(OutputPath)Microsoft.CSharp.targets" />
+    <!-- <BinX86Output Include="$(OutputPath)Microsoft.Data.Entity.targets" /> -->
+    <!-- <BinX86Output Include="$(OutputPath)Microsoft.DiaSymReader.Native.amd64.dll" /> -->
+    <!-- <BinX86Output Include="$(OutputPath)Microsoft.DiaSymReader.Native.x86.dll" /> -->
+    <BinX86Output Include="$(OutputPath)Microsoft.NetFramework.CurrentVersion.props" />
+    <BinX86Output Include="$(OutputPath)Microsoft.NetFramework.CurrentVersion.targets" />
+    <BinX86Output Include="$(OutputPath)Microsoft.NetFramework.props" />
+    <BinX86Output Include="$(OutputPath)Microsoft.NetFramework.targets" />
+    <!-- <BinX86Output Include="$(OutputPath)Microsoft.ServiceModel.targets" /> -->
+    <!-- <BinX86Output Include="$(OutputPath)Microsoft.VisualBasic.Core.targets" /> -->
+    <BinX86Output Include="$(OutputPath)Microsoft.VisualBasic.CurrentVersion.targets" />
+    <BinX86Output Include="$(OutputPath)Microsoft.VisualBasic.targets" />
+    <!-- <BinX86Output Include="$(OutputPath)Microsoft.WinFx.targets" /> -->
+    <!-- <BinX86Output Include="$(OutputPath)Microsoft.WorkflowBuildExtensions.targets" /> -->
+    <!-- <BinX86Output Include="$(OutputPath)Microsoft.Xaml.targets" /> -->
+    <BinX86Output Include="$(OutputPath)MSBuild.exe" />
+    <BinX86Output Include="$(OutputPath)MSBuild.exe.config" />
+    <!-- <BinX86Output Include="$(OutputPath)MSBuild.rsp" /> -->
```

Comment:
Yes, there are also some shim files we have in OSS that are shipped but named differently. Will have to fix that up as well.